src/datadog_directive.cpp

```diff
@@ -2,22 +2,74 @@
 
 #include <cassert>
 
+#include "datadog_conf_handler.h"
+
 namespace datadog {
 namespace nginx {
 
-char *warn_deprecated_command_datadog_tracing(ngx_conf_t *cf,
-                                              ngx_command_t * /*command*/,
-                                              void * /*conf*/) noexcept {
+char *silently_ignore_command(ngx_conf_t *cf, ngx_command_t *command, void *) {
+  ngx_conf_log_error(NGX_LOG_DEBUG, cf, 0, "Directive \"%V\" ignored",
```

Comment:
It is silent in terms of logging the error but let the program continue, right ?

Reply:
Auto answer: if the DEBUG_LOG are something we need to activate you can ignore this

Reply:
ditto